src/rascal/utils.hh

```diff
@@ -356,6 +356,12 @@ namespace rascal {
     std::string get_filename_extension(const std::string & filename);
 
   }  // namespace internal
+  constexpr int ThreeD{3};
```

Comment:
Are these used anywhere?

Reply:
Yes - and it is a start to use them everywhere else. Esp. in our current behler-dev branch. I'll leave it.

Reply:
Ok. I could not find a use in the gihtub diff, hence my question